dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/preheat/PreheatServiceTest.java

```diff
@@ -125,23 +124,6 @@ protected void setUpTest()
         userService = _userService;
     }
```

Comment:
Would it be good to add some additional test to cover the changes or are we well covered already?

Reply:
We are not too bad covered, but I can add a few more